src/test/java/com/salesforce/dynamodbv2/UpdateTest.java

```diff
@@ -122,56 +156,77 @@ void updateConditionalOnHkSuccess(TestArgument testArgument) {
         });
     }
 
-
     @ParameterizedTest(name = "{arguments}")
     @ArgumentsSource(DefaultArgumentProvider.class)
-    void updateConditionalOnGsiHkSuccess(TestArgument testArgument) {
+    void updateConditionalOnHkSuccessWithLiterals(TestArgument testArgument) {
```

Comment:
The diff is weird because I swapped this test case with updateConditionalOnGsiHkSuccess() so the former is adjacent to the related test case updateConditionalOnHkSuccess()...